packages/examples/package.json

```diff
@@ -7,6 +7,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
+    "lint": "exit 0",
```

Comment:
why is this needed for an example, what does it have to do with the main library under release?

Reply:
When I edited the `prePublish` command in `packages/express-oauth2-jwt-bearer` to prevent the tests being run twice I moved the `lint` command to run on the whole monorepo which requires every package to have a `lint` script (or it fails)